h2/src/test/scala/org/scalarelational/h2/modular/ModularSpec.scala

```diff
@@ -158,7 +200,36 @@ object ModularDatastore extends H2Datastore {
   object users extends Table("users") with ModularSupport {
     val name = column[String]("name", Unique)
     val age = column[Int]("age")
+    val created = column[Option[Timestamp]]("created")
     val modified = column[Option[Timestamp]]("modified")
     val id = column[Int]("id", PrimaryKey, AutoIncrement)
   }
-}
\ No newline at end of file
+
+  // If multiple classes should be equipped with `created` and `modified` fields,
+  // then this mixin can be used.
+  trait Timestamps extends ModularSupport {
+    // TODO How can we get rid of this method?
+    def column[T](name : String, properties : ColumnProperty*)
+                 (implicit converter : DataType[T], manifest : Manifest[T]): ColumnLike[T]
+
+    val created  = column[Timestamp]("created")
+    val modified = column[Timestamp]("modified")
```

Comment:
This trait is not part of ScalaRelational. It's just an example. It's actually more convenient to have a trait like this because you don't need to define ``created`` and ``modified`` in all tables yourself.